Makefile

```diff
@@ -205,10 +205,10 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found -f -
 
 deploy: kustomize edit-image ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/begin | kubectl apply -f -
+	./deploy.sh deploy $(KUSTOMIZE) config/begin
```

Comment:
On the "deploy" lines, the "config/begin" isn't used because it's hard coded into the deploy.sh script. Does it have to be included on this line? Same for all the other PRs too.

Reply:
I hadn't meant for it to be hardcoded.  I'll adjust that.